include/fty_common_db_asset.h

```diff
@@ -134,6 +139,13 @@ namespace DBAssets {
                                  std::string status,
                                  std::function<void(const tntdb::Row&)> cb);
 
+// select_asset_element_by_dc: select everything under a specified DC from v_web_element
+// returns -1 in case of error or 0 for success
+    int
+    select_asset_element_by_dc (tntdb::Connection& conn,
+                                int64_t dc_id,
+                                std::function<void(const tntdb::Row&)>& cb);
```

Comment:
I'm not familiar with `std::function`, but shouldn't `cb` be either a *const* reference or a value here?

Reply:
Added const, seems to work okay :-)

Reply:
Using const reference won't allow lambdas being passed directly to function as temp objects. I'm not really sure we're using this but there's always a chance. I'm also in favor to keep current approach as for other similar functions to use value instead of const reference, since it can also benefit of std::move

Reply:
std::function has low footprint. References are not necessary.